src/modules/map/hooks/use-mapbox-json-style.tsx

```diff
@@ -6,6 +6,35 @@ import {useFirestoreConfigurationContext} from '@atb/modules/configuration';
 import {getTextForLanguage, useTranslation} from '@atb/translations';
 import {useVehiclesAndStationsVectorSource} from '../components/mobility/VehiclesAndStations';
 
+// since layerIndex doesn't work in mapbox, but aboveLayerId does, add some slot layer ids to use
+export enum MapSlotLayerId {
+  GeofencingZones = 'geofencingZones',
+  Vehicles = 'vehicles',
+  Stations = 'stations',
+  NSRItems = 'nsrItems',
+  SelectedFeature = 'selectedFeature',
+}
+
+const slotSourceKey = 'slotSource';
+// This source only exists for slots layers, no data is fetched.
+const slotSource: StyleJsonVectorSourcesObj = {
+  [slotSourceKey]: {type: 'vector'}, // type is required, but otherwise doesn't matter here.
```

Comment:
```suggestion
  [slotSourceKey]: {type: 'shape'}, // type is required, but otherwise doesn't matter here.
```
@tormoseng after merging this, let's make sure there are no network calls to "nowhere" because of this. If so, we should make this change.